include/openssl/ocsp.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2015-2021 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+
+#ifndef AWSLC_OCSP_H
+#define AWSLC_OCSP_H
+
+#pragma once
+
+#include <openssl/asn1t.h>
+#include <openssl/x509.h>
+#include <openssl/x509v3.h>
+#include <openssl/safestack.h>
+
```

Comment:
You must declare these definitions and function declarations `extern "C"`. Can be done file-globally: see https://github.com/awslabs/aws-lc/blob/main/include/openssl/asn1.h#L70